### PR TITLE
docs(hub) introduce decK

### PR DESCRIPTION
**Pull request by hbagdi**
Add [decK](https://github.com/hbagdi/deck) to Kong Hub.

### CODE REVIEW

**Comment by hbagdi** [comment]
I couldn't get the site running locally due to an issue with bundle.
I'm hoping the netlify deploy will help us make sure the new content renders correctly before merge. Thanks!

File: app/_hub/hbagdi/deck/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+    - 1.x
+    incompatible:
+    - <= 0.14.x
+  enterprise_edition: # optional
+    compatible:
+    - >= 0.35.x
```
Comment (levfishbluefish) [inline app/_hub/hbagdi/deck/index.md]: This line is preventing the file from compiling and showing up on the Hub list

File: app/_hub/hbagdi/deck/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+publisher: hbagdi
+
+categories:
+- deployment
+
+type: integration
+
+desc: Backup, restore, diff and sync configuration to Kong
+description: |
+  decK helps manage Kong's configuration in a declaratice fashion.
```
Comment (levfishbluefish) [inline app/_hub/hbagdi/deck/index.md]:
```suggestion
  decK helps manage Kong's configuration in a declarative fashion.
```

**Comment by hbagdi** [comment]
@levfishbluefish Do you know why the decK doesn't actually show up in the preview of this PR?
https://deploy-preview-1418--kongdocs.netlify.com/hub/

**Comment by hbagdi** [comment]
NVM, just saw the comment in the review.

**Comment by hbagdi** [comment]
Okay, so it shows up in the preview now:
https://deploy-preview-1418--kongdocs.netlify.com/hub/hbagdi/deck/

Thanks @levfishbluefish!

**Pull request merged**